src/plone/passwordless/browser/views.py

```diff
@@ -0,0 +1,9 @@
+from Products.CMFPlone.resources import add_resource_on_request
+from Products.Five import BrowserView
+
+class PracticingView(BrowserView):
+
+    def __call__(self):
```

Comment:
We don't need  the `__call__()` implementation. Remove it and add a `pass` statement to the class.